tmt-app/src/Components/Questions.jsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { Container, Grid } from "@mui/material";
+import AnatomyIcon from "../Images/icons/anatomy.svg";
+import QuestionCard from "./QuestionCard";
+
+function Questions() {
+  return (
+    <Container>
+      <Grid container justifyContent="center">
+        {TEST_ITEMS.map((item) => (
+          <Grid item key={item.title}>
+            <QuestionCard item={item} />
+          </Grid>
+        ))}
+      </Grid>
+    </Container>
+  );
+}
+
+export default Questions;
+
+const TEST_ITEMS = [
```

Comment:
Same feedback Ive left for Raymond, let's export this from a mock data file so we can clean up the components we're actually importing them from